src/examples/example-controls.tsx

```diff
@@ -1,15 +1,24 @@
 import { Component, h } from '@stencil/core';
 
+/**
+ * This component is only used in our documentations
+ * to provide a container for settings of examples.
+ *
+ * For example, it visually groups and organizes checkboxes
+ * used to show different states of components,
+ * such as Disabled, Required, Readonly, etc…
+ *
+ * :::warning
+ * For internal use only!
+ * :::
+ */
 @Component({
     tag: 'limel-example-controls',
+    shadow: true,
     styleUrl: 'example-controls.scss',
 })
 export class ExampleControls {
     public render() {
-        return (
-            <limel-flex-container justify="end">
-                <slot />
-            </limel-flex-container>
-        );
+        return <slot />;
```

Comment:
I think this commit (`feat(example-controls): make it more flexible and useful`) should be a `chore` or `refactor`. It doesn't affect the production code, so shouldn't trigger a new version. It could be a `docs`, if it makes a big enough change for the reader of the docs, that it's worth telling them about it. Basically, if it's something like "this thing was a significant problem when using the docs, and now we've fixed it, so you might want to have another look if there was something you couldn't do or understand before".

Reply:
Ok, sounds right. I was thinking maybe adding the new CSS variables makes it a `feat` for "us". But since it's a docs thing, perhaps better not to use that, and as you say use `chore` or `docs`. In this case I myself vote for `docs`.

Reply:
⚡ changed to `docs`

Reply:
`feat` and `fix` only applies to to changes that affects either the consumer or the user 🙂